pkgs/applications/misc/kiwix/default.nix

```
@@ -9,15 +9,15 @@
 , aria2
 }:
 
-mkDerivation rec {
+mkDerivation {
   pname = "kiwix";
-  version = "2.3.1";
+  version = "unstable-2024-02-20";
```

```suggestion
  version = "2.3.1-unstable-2024-02-20";
```
https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-naming